luminoso_api/cli.py

```diff
@@ -0,0 +1,123 @@
+import argparse
+import csv
+import json
+import os
+import sys
+from signal import signal, SIGPIPE, SIG_DFL
+from urllib.parse import urlparse
+
+from .client import LuminosoClient
+from .constants import URL_BASE
+from .errors import LuminosoError
+
+# Python raises IOError when reading process (such as `head`) closes a pipe.
+# Setting SIG_DFL as the SIGPIPE handler prevents this program from crashing.
+signal(SIGPIPE, SIG_DFL)
+
+
+DESCRIPTION = "Access the luminoso api via the command line."
```

Comment:
One capitalization thing I didn't notice before: "luminoso api" should be "Luminoso API".